ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -108,7 +108,8 @@ class EditMode {
                     },
                     row: {
                         enabled: true
-                    }
+                    },
+                    settingsButtonEnabled: true
```

Comment:
I'm not convinced if this should be here...

IMO in the `toolbar` we have options for the toolbar itself.
The setting button, at least for me, is something like the `dragDrop` or `resize` and it should have a separate config object like:
```js
                dragDrop: {
                    enabled: true
                },
                resize: {
                    enabled: true
                },
                settings: {
                    enabled: true
                },
```

Altho I can imagine in the future someone would like to disable separately for the cell/row...
But let's not get too far 😅